_schemaV2.graphql

```diff
@@ -22380,27 +22333,23 @@ input ViewingRoomSubsectionInput {
   internalID: ID
 }
 
-# The connection type for ViewingRoom.
+# A connection to a list of items.
 type ViewingRoomsConnection {
   # A list of edges.
   edges: [ViewingRoomsEdge]
-
-  # A list of nodes.
-  nodes: [ViewingRoom]
-  pageCursors: PageCursors
+  pageCursors: PageCursors!
```

Comment:
there was no `!` before - breaking change?

Reply:
i dont think so! as long as the MP pagination resolvers dont generate any nulls (they shouldn't)